Source/Core/DolphinTool/ExtractCommand.cpp

```diff
@@ -97,6 +97,20 @@ static void ExtractPartition(const DiscIO::Volume& disc_volume, const DiscIO::Pa
   ExtractSystemData(disc_volume, partition, out);
 }
 
+static void ListRecursively(std::string path, const DiscIO::FileInfo& info,
+                            std::string* result_text)
+{
+  fmt::println("{}", path);
+  result_text->append(path);
```

Comment:
You're no longer adding a newline when appending to `result_text`.

Reply:
Unless I'm mistaken, the newline is added here using `println` instead.

Reply:
Yes, there is a newline printed to the console, but no newline is added to `result_text`.

Reply:
Oh my bad, I missed the fact that `result_text` was an output parameter.

Reply:
Thanks for catching this!